test/xpu_api/utilities/meta/meta.trans/meta.trans.other/common_type.pass.cpp

```diff
@@ -170,7 +170,9 @@ typedef void (S::*PMF)(long) const;
 typedef char S::*PMD;
 
 using dpl::is_same;
+#    if TEST_STD_VER == 17
 using dpl::result_of;
+#    endif // TEST_STD_VER
```

Comment:
It seems like `result_of` is never used in this file outside of this place. I suggest to remove it.
```suggestion
```